README.md

```diff
@@ -123,7 +123,29 @@ This will ensure all event handlers will be registered rather being overruled by
 
 ### useRole
 
-(tbd)
+#### Usage
+
+```html
+<script>
+	import { useFloating, useInteractions, useRole } from '@skeletonlabs/floating-ui-svelte';
+
+	const floating = useFloating();
```

Comment:
Remove the extra line breaks between `const` declarations to save vertical space.